benchmarks/ALIA/sv-benchmarks/2019.array-cav/array_tiling_poly6.sl

```diff
@@ -0,0 +1,32 @@
+(set-logic ALIA)
+
+(synth-inv inv_fun ((a (Array Int Int)) (i Int) (s Int)))
+
+(declare-primed-var a (Array Int Int))
+(declare-primed-var i Int)
+(declare-primed-var s Int)
+
+(define-fun pre_fun ((a (Array Int Int)) (i Int) (s Int)) Bool
+  (and (= i 0) (> s 1)))
+
+(define-fun trans_fun ((a (Array Int Int)) (i Int) (s Int) (a! (Array Int Int)) (i! Int) (s! Int)) Bool
+ (and (< i (* 2 s))
+       (= i! (+ i! 1))
+       (= s! s)
+       (ite (< i s)
+                    ((= a! (store a i (* (- i 1) (+ i 1)))))
+                    ((= a! (store a (- i s) (- (select a (- i s)) (* (- i s) (- i s))))))
+       )
```

Comment:
This translation doesn't look correct.

Is this the source file: https://github.com/sosy-lab/sv-benchmarks/blob/master/c/array-cav19/array_tiling_poly6.c?

Reply:
Yes, that is the source file. I wanted to use one transition to combine the two loops? Let me know which part doesnt look good?

Reply:
I see, then the file makes sense 👍